src/views/subscribe/style.css

```diff
@@ -0,0 +1,224 @@
+.heading-class {
+    width: 81%;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 1.2;
+    margin-bottom: 50px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+
+}
+span.svg-wrapper {
+    position: absolute !important;
+    bottom: -6px;
+    margin-left: 6px;
+}
+.subscribe-class span {
+    font-family: helvetica;
+}
+
+.points-class {
+    font-size: 18px;
+    margin-left: 7px;
+}
+
+.sub-span-heading {
+    font-size: 18px;
+}
+
+.note-class {
+    margin-top: 75px;
+}
+
+.form-group label {
```

Comment:
please remove this, since there is already  `.form-group label` defined at line 164

Reply:
please remove all the duplicate styling.